storageClient/api.go

```diff
@@ -73,12 +73,22 @@ type AddingCapabilityRequest struct {
 	AccountID      uuid.UUID `json:"account_id"`
 }
 
-// AddingClientToGroupRequest wraps values used to add a client to a Group.
-type AddingClientToGroupRequest struct {
-	ClientID       uuid.UUID `json:"client_id"`
-	Name           string    `json:"group_name"`
-	CapabilityName string    `json:"capability_name"`
-	AccountID      uuid.UUID `json:"account_id"`
+// NewMember wraps values needed to add a client to a Group.
+type NewMember struct {
+	ClientID       []uuid.UUID `json:"client_ids"`
```

Comment:
When will a member have multiple client_ids?

Reply:
oops! missed that one. It was when i originally wasnt doing GroupMember and the Group member array